NorthstarDedicatedTest/logging.cpp

```diff
@@ -407,6 +407,86 @@ bool CClientState_ProcessPrint_Hook(__int64 thisptr, __int64 msg)
 	return true;
 }
 
+
+typedef double(__fastcall* GetTimeConnectedType)(__int64 thisptr);
+GetTimeConnectedType GetTimeConnected;
+
+typedef char*(__fastcall* GetAddressType)(__int64 thisptr);
+GetAddressType GetAddress;
+
+const char* COM_FormatSeconds(int seconds)
+{
+	static char string[64];
+
+	int hours = 0;
+	int minutes = seconds / 60;
+
+	if (minutes > 0)
+	{
+		seconds -= (minutes * 60);
+		hours = minutes / 60;
+
+		if (hours > 0)
+		{
+			minutes -= (hours * 60);
+		}
+	}
+
+	if (hours > 0)
+	{
+		snprintf(string, sizeof(string), "%2i:%02i:%02i", hours, minutes, seconds);
+	}
+	else
+	{
+		snprintf(string, sizeof(string), "%02i:%02i", minutes, seconds);
+	}
```

Comment:
Having it consistently print `hh:mm:ss` would be preferred in terms of parsing over changing format in case the server has been online <1h

Reply:
(Function was copied from leaked CS:GO source so I guess we're keeping it this way.)